### DISCUSSION TITLE
Project Name Idea

### DISCUSSION THREAD

**Discussion by jocko96b** [complexlogic/big-launcher#12] [General]
Freestyle is my suggestion, for two reasons:

1. The app allows customization for whatever the user needs, supporting any type of application, and can run on any OS.
2. It was the original code name for what became Windows Media Center back for Windows XP.

I also threw some stuff into DALL-E for an icon and found one that looked neat enough:

![Freestyle](https://github.com/complexlogic/big-launcher/assets/39993614/e71a4ffc-8cf5-4d69-a155-79c2167ab072)

**Comment by abn0mad** [comment]
Great project; deserving of much wider attention. I am eagerly awaiting the completion of this project, I use flex-launcher daily, however, and I could not be more pleased. Thank you for starting-, and for all your magnificent work on both projects. :)

Given that this launcher is less customisable than flex-launcher (according to its developer), and that I believe that the name should reflect its function (in terms of being easy to find for potential users):

* media-launcher
* media-home
* htpc-home
* SmarTVY (a play on smart TV and DIY)
* MediaHub
* DIYTV

I reckon SmarTVY is a little nifty.

Again, many thanks for both flex-launcher and this project, fantastic initiative and work :)

**Comment by jhnpldng01** [comment]
Launch All

I'd like the TV to start up with the look of a Smart TV full screen menu but not be limited to typical Smart TV or HTPC apps.

Additional menu items for me would be;

Apps
-- Terminal
-- LibreWolf 
-- Falkon
-- Qbittorrent
-- ZimaOS (or other NAS Dashboard)

hence Launch All

Thought of kind of a hokey one; Go for Launch

Either one could have a rocket as an icon, although rockets as icons/logos are a bit overused already. They all seem to be short, fat rounded cartoonish things though . Probably to fit in a circle/square. The computer age killed logo design with the circle/square icon restriction.

Launch Pad is taken. Very literate would be Launch Screen or Full Launch Screen.
